src/app/lib/utilities/getUrlHelpers/index.js

```diff
@@ -0,0 +1,17 @@
+import getEmbedUrl from './getEmbedUrl';
+import getMasterbrand from '../getMasterbrand';
```

Comment:
I know you're already looking into this, @Louis-Matsika, but for the benefit of others reviewing we are:
- moving non-URL helpers back to their original location
- importing URL helpers using the new index.js file instead of from folders deeper in the tree

Reply:
@Louis-Matsika I believe `getMasterbrand`, `getMediaId`, and `getStoryPromoInfo` need removing from this file as they appear to have been moved back to where they were before?

cc @simonsinclair